packages/synapse-react-client/src/components/DataGrid/components/GridMenuButton/GridMenuButton.module.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../../../style/abstracts/variables' as SRC;
+@use 'sass:map';
+
+.button {
+  height: 36px;
+  font-size: 16px;
+  font-weight: 400;
+  white-space: nowrap;
+
+  &:global(.MuiButton-outlined:not(.Mui-disabled)) {
```

Comment:
The `:global` selector means "do not transform this classname!". In this case, we target MUI's classnames to override their styles.